installation/manual-installation/mongo-versions/README.md

```diff
@@ -0,0 +1,7 @@
+# Supported Mongo Versions
+
+Rocket.Chat currently requires MongoDB from version 2.3 and onwards.
+
+## Supported Storage Engines
+
+The Storage engine required for Rocket.Chat is [MMAPv1](https://docs.mongodb.com/manual/core/mmapv1/).
```

Comment:
We actually are trying to change this.  We are running wiredtiger behind open.rocket.chat.  But current recommendation is mmapv1 for large installs.  That might change depending on findings